framework/src/node/state_machine/constants.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+export const EVENT_INDEX_INIT_GENESIS_STATE = Buffer.from([0]);
+export const EVENT_INDEX_FINALIZE_GENESIS_STATE = Buffer.from([1]);
+
+export const EVENT_INDEX_BEFORE_TRANSACTIONS = Buffer.from([2]);
+export const EVENT_INDEX_AFTER_TRANSACTIONS = Buffer.from([3]);
```

Comment:
@ricott1 I added the genesis prefix like this for now. Please let me know if other constants are better.

Reply:
these are fine, I will update the LIP when I open the PR on lips